jobs/JGLOBAL_ARCHIVE

```diff
@@ -14,16 +14,15 @@ YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_ANALYSIS COM_ATMOS_BUFR COM_ATMO
   COM_ATMOS_TRACK COM_ATMOS_WMO \
   COM_CHEM_HISTORY COM_CHEM_ANALYSIS\
   COM_MED_RESTART \
-  COM_ICE_HISTORY COM_ICE_INPUT COM_ICE_RESTART \
+  COM_ICE_HISTORY COM_ICE_INPUT COM_ICE_RESTART COM_ICE_GRIB \
```

Comment:
@aerorahul, do you want to archive COM_ICE_NETCDF?

Reply:
I agree since we are archiving the history files. I asked because COM_OCEAN_NETCDF is archived. Just wondering why ocean but not ice.

Reply:
Here, just the variables are being defined.  The archival logic is in `hpssarch_gen.sh` and I confirmed they are not being archived.

Reply:
Great, thanks!